### PR TITLE
Removing duplicate date formatting for listing expiry date in listing dashboard

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

It removes duplicate date formatting code for listing expiry date in listing dashboard

## QA Instructions, Screenshots, Recordings

expiry_date

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by Rafi993** [comment]
I did not fill in the category for listing. That is what shows up extra space in the above screenshot

File: app/javascript/listings/dashboard/listingRow.jsx [hunk 1]
```diff
@@ -42,7 +36,12 @@ export const ListingRow = ({ listing }) => {
         bumpedAt={listing.bumped_at}
         updatedAt={listing.updated_at}
       />
-      {listingExpiry}
+      {expiryDate && (
+        <Fragment>
+          {' '}
+          | Expires on: <DateTime dateTime={expiryDate} />
```
Comment (nickytonline) [inline app/javascript/listings/dashboard/listingRow.jsx:42]:
```suggestion
          {' | Expires on: '}<DateTime dateTime={expiryDate} />
```

**Review by nickytonline — APPROVED** [review]
Small suggestion, but otherwise good to go @Rafi993!

**Review by nickytonline — APPROVED** [review]

File: app/javascript/listings/dashboard/listingRow.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import PropTypes from 'prop-types';
-import { h } from 'preact';
+import { h, Fragment } from 'preact';
```
Comment (vaidehijoshi) [inline app/javascript/listings/dashboard/listingRow.jsx:2]: Nice! I'm reacquainting myself with React (and learning more about Preact!) and I hadn't heard of `Fragment` components before! Pretty neat; thanks for teaching me about them by using them in your PR 😸

**Review by vaidehijoshi — APPROVED** [review]

**Comment by Rafi993** [comment]
Thanks @rhymes

**Pull request merged**